WeakAurasOptions/ConditionOptions.lua

```diff
@@ -438,12 +438,138 @@ local function addControlsForChange(args, order, data, conditionVariable, condit
+        and conditions[i].changes[j].value[1]
+        and conditions[i].changes[j].value[2]
+        and conditions[i].changes[j].value[3]
+        and conditions[i].changes[j].value[4]
+         ) then
+          return conditions[i].changes[j].value[1], conditions[i].changes[j].value[2], conditions[i].changes[j].value[3], conditions[i].changes[j].value[4];
+        else
+          return 0.95, 0.95, 0.32, 1
+        end
+      end,
+      set = setValueColor,
+      hidden = function()
+        return type(conditions[i].changes[j].value) == "table" and conditions[i].changes[j].value.glow_action == "hide" 
+      end 
+    }
+    order = order + 1;
+--[[ -- Glow picker frame conditions (WIP)
+    args["condition" .. i .. "value" .. j .. "glow_frame"] = {
+      type = "input",
+      width = WeakAuras.doubleWidth*.7,
+      name = blueIfNoValue2(data, conditions[i].changes[j], "value", "glow_frame", L["Frame"], L["Frame"]),
+      desc = descIfNoValue2(data, conditions[i].changes[j], "value", "glow_frame", propertyType),
+      order = order,
+      get = function()
+        return type(conditions[i].changes[j].value) == "table" and conditions[i].changes[j].value.glow_frame;
+      end,
+      set = setValueComplex("glow_frame"), 
+      hidden = function()
+        return type(conditions[i].changes[j].value) == "table" and conditions[i].changes[j].value.glow_action == "hide" 
+      end 
+    }
+    order = order + 1;
+
+    args["condition" .. i .. "value" .. j .. "glow_frame_picker"] = {
+      type = "execute",
+      width = WeakAuras.doubleWidth*.3,
+      name = blueIfNoValue2(data, conditions[i].changes[j], "value", "glow_frame_picker", L["Pick"], L["Choose Frame"]),
+      desc = descIfNoValue2(data, conditions[i].changes[j], "value", "glow_frame_picker", propertyType, L["hoose Frame"]),
+      order = order,
+      func = function()
+        if type(data.conditions[i].changes[j].value) == "table" and data.conditions[i].changes[j].value.frame_input then
+          --WeakAuras.StartFrameChooser(WeakAuras.GetData(data.conditions[i].changes[j].value.frame_input), {"conditions", i, "changes", j, "value", "frame_input"});
+          print(WeakAuras.GetData(data.conditions[i].changes[j].value.frame_input))
+        else
+          WeakAuras.StartFrameChooser(data.conditions[i].changes[j].value, {"frame_input"})--{"conditions", i, "changes", j, "value", "frame_input"});
+        end
+      end, 
+      hidden = function()
+        return type(conditions[i].changes[j].value) == "table" and conditions[i].changes[j].value.glow_action == "hide" 
+      end 
+    }
+    order = order + 1;
+      elseif (propertyType == "animate") then 
+    local animation_types = {
+      pulse = L["Pulse"],
+      bounce = L["Bounce"],
+      flip = L["Filp"],
+      shake = L["Shake"],
+      spin = L["Spin"],
+      spiral = L["Spiral"],
+    }
+    args["condition" .. i .. "value" .. j .. "animation_type"] = {
```

Comment:
Animation??? Does this really belong in this pull request?

Conditional animations are a much bigger scope then this, I think.